src/wallet/api/wallet2_api.h

```diff
@@ -40,6 +40,9 @@
 #include <stdexcept>
 #include <cstdint>
 
+#include "cryptonote_basic/cryptonote_basic.h"
```

Comment:
We shouldn't include these headers in the wallet API head since this API should "stand on its own". Notice that the only other headers included above are standard C++.